002/002.go

```diff
@@ -1,26 +1,29 @@
 package main
 
-import "fmt"
+import (
+    "fmt"
+)
 
 func main() {
-	var t int
-	fmt.Scanf("%d", &t)
-	for i := 0; i < t; i++ {
-		var n int
-		fmt.Scanf("%d", &n)
-
-		a := 1
-		b := 2
-		var c, sum int
-		for a < n {
-			if a%2 == 0 {
-				sum += a
-			}
-			c = a + b
-			a = b
-			b = c
-		}
-
-		fmt.Println(sum)
-	}
-}
+    var t int
+    fmt.Scanf("%d", &t)
+    if t >= 1 && t <= 100000 {
```

Comment:
Please make sure that you add the constraints for the variable t and n given in the problem statement.